README.md

```diff
@@ -90,13 +90,13 @@ You can start a new Camel K integration with or without additional options such
 
     The "Basic" option starts the file as a new integration to deploy the integration to the running Kubernetes system.
 
-* **ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap**
+* **ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap as runtime configuration**
 
     Select a ConfigMap from a list of the published ConfigMaps in your current Kubernetes system. For example:
 
     ![Start Apache Camel K Integration - ConfigMap list](images/kubernetes-configmap-list.jpg)
 
-* **Secret - Apache Camel K Integration with Kubernetes Secret**
+* **Secret - Apache Camel K Integration with Kubernetes Secret as runtime configuration**
```

Comment:
Would adjust capitalization of the new text to match the previous text: "as Runtime Configuration"